src/GlobalStyles.js

```diff
@@ -9,9 +9,11 @@ export default createGlobalStyle`
     --primary-background: #A5A697;
     --secondary-background: #F2F2F2;
     --primary-text: #080808;
+    --secondary-text:#323331;
     --primary-border: #F2F2F2;
     --primary-shadow: #8c8d80;
     --secondary-shadow: #bebfae;
+    --third-shadow: #757670;
 
     --primary-black: #0D0D0D;
     --primary-anthrazit: #262626;
```

Comment:
I think it's best practice to stick to the same pattern when naming styling variables:
- maybe you find different expressions for `--primary-black` and `--primary-anthrazit` (that don't actually include the color)

Reply:
this mean the general Color. I thing about it.